src/main/java/com/uber/cadence/serviceclient/WorkflowServiceTChannel.java

```diff
@@ -339,6 +345,8 @@ private <T> T measureRemoteCallWithTags(
       sw.stop();
       scope.counter(MetricsType.CADENCE_ERROR).inc(1);
       throw e;
+    } finally {
+      span.finish();
```

Comment:
out of curiuosity, is there any kind of error attach you can put on the span to indicate the type of error? I vaguely seem to recall that was possible

Reply:
ah, it's a tag: https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table

Reply:
Yes, we can also add logs.